src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "White";
+    private static final int DEFAULT_RADIUS = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(6);
+        String color = colorSupplier.getRandomColor();
+        switch (index) {
+            case 0:
+                return createRandomSquare(color);
+            case 1:
+                return createRandomRectangle(color);
+            case 2:
+                return createRandomTriangle(color);
+            case 3:
+                return createRandomCircle(color);
+            case 4:
+                return createRandomTrapezoid(color);
+            default:
+                return getDefaultFigure();
+        }
+    }
+
+    public Square createRandomSquare(String color) {
```

Comment:
all methods for creating figure should be private and after public methods

There should be only two public methods getRandomFigure() and getDefaultFigure()
after them should go privte ones